isis/tests/FunctionalTestsGrid.cpp

```diff
@@ -263,7 +263,15 @@ TEST_F(NewHorizonsCube, FunctionalTestGridBandDependent) {
 TEST_F(DefaultCube, FunctionalTestGridExtend) {
   QVector<QString> args = {"to=" + tempDir.path() + "/output.cub", "extendgrid=true"};
   UserInterface options(APP_XML, args);
-  grid(testCube, options);
+
+  // change mapping group of cube to one that extends past longitude domain
+  Pvl newMap;
+  newMap.read("data/defaultImage/extendProj.map");
+  PvlGroup &newMapGrp = newMap.findGroup("Mapping", Pvl::Traverse);
+
+  projTestCube->putGroup(newMapGrp);
```

Comment:
In general, in the past, we have used references to PVL instances to change labels. Cube labels are pointers and you can retrieve PVL Objects and Groups as references to change in place. But, if it works it works.